src/scripts.js

```diff
@@ -11,13 +11,13 @@ console.log('This is the JavaScript entry file - your code begins here.');
 
 // An example of how you tell webpack to use a JS file
 
-import userData from './data/users';
+fetch('https://pacific-badlands-43237.herokuapp.com/api/v1/users')
+.then(req => req.json())
+.then(data => renderUser(data.userData));
```

Comment:
Using the renderUser() within the callback is awesome! Wouldn't have thought of this!